tutorials/SpectroscopicDataReductionBasics/1-SpectroscopicTraceTutorial.ipynb

```diff
@@ -0,0 +1,1100 @@
+{
```

Comment:
<p>I've never seen this <code>_=</code>  thing before. Could you say more about what it is and why it's useful in this situation. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/508/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I've changed the style to use ;'s at the end of the line and added an explanation.  Thanks @adrn for pointing out this shorthand.</p>